peps/pep-0751.rst

```diff
@@ -177,6 +199,41 @@ consistent order. Usage of inline tables SHOULD also be kept consistent.
   (i.e. the minimum viable Python version for the lock file).
 
 
+``extras``
+==========
+
+- **Type**: Array of strings
+- **Required?**: no; defaults to ``[]``
+- **Inspiration**: :ref:`packaging:pyproject-tool-table`
```

Comment:
Is that the correct inspiration key for extras and dependency groups?

Reply:
I don't think it's critical, but it was for me. I could change it to core metadata.